src/components/views/__tests__/CinemaHall.spec.js

```diff
@@ -0,0 +1,54 @@
+import { describe, it, expect } from "vitest";
+import { shallowMount } from "@vue/test-utils";
+import CinemaHall from "@/components/views/CinemaHall.vue";
+import rowLetter from "@/components/views/CinemaHall.vue";
+import isSeatTaken from "@/components/views/CinemaHall.vue"
+
+
+const createComponent = () =>
+  shallowMount(CinemaHall, {
+    props: { rows, seats, seatsTaken },
+  });
+
+const rows = 10;
+const seats = 10;
+const seatsTaken = ['A1'];
+
+describe("CinemaHall.vue", () => {
+  it("expects to be mounted", () => {
+    const wrapper = createComponent();
+    expect(wrapper).toBeTruthy();
+  });
+
+  it("should display a given seats number in a row", () => {
+    const wrapper = createComponent();
+    expect(wrapper.findAll('.cinema-hall__row').length).toBe(seats)
+  });
+
+  it("should display a given number of rows", () => {
+    const wrapper = createComponent();
+    expect(wrapper.findAll('[data-spec="row"]').length).toBe(rows)
+  });
+
+  it("should generate letters for rows numbers", () => {
+    const wrapper = shallowMount(rowLetter)
+    expect(wrapper.vm.rowLetter(3)).toBe('C')
```

Comment:
you are testing the internal method of a component not **the output** (which is wrong!) - instead try to find the place where it's displayed and check if it shows the correct text in the HTML 💡